doc/develop/west/without-west.rst

```diff
@@ -69,6 +69,10 @@ If you don't have west installed and your application *does* need one
 of these repositories, you must set :makevar:`ZEPHYR_MODULES`
 yourself as shown above.
 
+The implementation CMake uses to set :makevar:`ZEPHYR_MODULES` can be
+found in the Zephyr source code in  :file:`cmake/modules/zephyr_module.cmake`.
```

Comment:
Thanks for this patch -- please use `:zephyr_file:` instead of `:file:`. This will generate a link to the correct version of the file for whatever version of zephyr this documentation ends up getting built for, etc.